### PR TITLE
Some updates to the 'getting started' vignette

### PR DESCRIPTION
**Pull request by florisvdh**
This PR is the follow-up to https://github.com/r-spatial/qgisprocess/pull/144#issuecomment-1547415768.

Various minor changes have been done in the vignette; can you have a look @jannes-m? Once approved, you can merge this one, as it will add to your PR.

This also installs the SAGA Next Gen plugin in all Ubuntu jobs, and uses the plugin presence as an extra condition for building the vignette. The pkgdown website should consequently render the vignette.

### CODE REVIEW

**Comment by jannes-m** [comment]
This looks great, thanks a lot! I hope to review your changes tomorrow in the evening.

File: man/vignette_childs/_qgisprocess.Rmd [hunk 1]
```diff
@@ -196,7 +207,7 @@ dem <- rast(system.file("raster/dem.tif", package = "spDataLarge"))
 info <- qgis_run_algorithm(alg = "grass7:r.slope.aspect", elevation = dem)
 ```
 
-Just printing the `info` object shows which output objects have been made:
+Just printing the `info` object shows which output files have been made:
```
Comment (jannes-m) [inline man/vignette_childs/_qgisprocess.Rmd:210]: Well spotted!

File: man/vignette_childs/_qgisprocess.Rmd [hunk 2]
```diff
@@ -247,7 +258,7 @@ rp_tp = qgis_run_algorithm(
 One could use the `list()` command but it is recommendended to use the `qgis_list_input()` function which is more robust, and therefore will also support non-JSON-input configurations (e.g. QGIS < 3.24).
 
 ```{r multilayer-list, eval=FALSE}
-rp_tp = qgis_run_algorithm(
```
Comment (jannes-m) [inline man/vignette_childs/_qgisprocess.Rmd:250]: old habits die hard, thanks for changing!

**Review by jannes-m — APPROVED** [review]
Thanks for updating the CI stuff and the vignette, @florisvdh. Since my changes to this PR are non-controversial, I think, I will merge this PR into my PR. And should you be unhappy with one of my changes, you can still revert it there.

**Comment by florisvdh** [comment]
Thanks @jannes-m! I spotted some more stuff; will add it to the `vignettes` branch.

**Pull request merged**